src/test/kotlin/uk/gov/dluhc/printapi/messaging/ProcessPrintRequestBatchMessageListenerIntegrationTest.kt

```diff
@@ -94,6 +94,71 @@ internal class ProcessPrintRequestBatchMessageListenerIntegrationTest : Integrat
         }
     }
 
+    @Test
```

Comment:
This test is identical to the one above except for the `isFromApplicationsApi` bit and the assert around the update statistics message, can we merge the two of them into a parameterised test instead please